cli.js

```diff
@@ -1,124 +1,146 @@
+import {bold, brightGreen, red} from "./deps/colors.js";
 import upgrade from "./cli/upgrade.js";
 import update from "./cli/update.js";
 import init from "./cli/init.js";
-import help from "./cli/help.js";
 import build from "./cli/build.js";
+import run from "./cli/run.js";
+
+export const version = "v0.14.0";
+
+const USAGE = `
+    Docs: https://lumeland.github.io/
+
+        To build the site:
+            lume
+
+        To serve the site in localhost
+            lume --serve
+
+        To update lume to the latest version
+            lume update
+
+        To run a custom script
+            lume run script-name
+
+        To get help with a command
+            lume COMMAND --help
+
+    USAGE:
+        lume [COMMAND] [OPTIONS]
+
+    COMMANDS:
+        build      Build the site. It's the default command
+        init       Creates a _config.js file
+        run        Run an site script
+        upgrade    Upgrade 🔥lume to the latest version
+        update     Update the version of the lume modules imported in a _config.js file
+
+
+    OPTIONS:
+        -h, --help              print usage information
+        -v, --version  Prints version information
+            --root <dir>        the root that lume should work in   Default: ./
+            --src  <dir>        the source directory for your site  Default: ./
+            --dest  <dir>       the build destination.              Default: _site
+            --config <file>     specify the lume config file.       Default: _config.js
+            --location <domain> set the domain for your site.       Default: http://localhost
+            --dev               enable dev mode (view draft pages)
+            
+            --serve             start a live-reloading web server
+            --port <port>       the port the server is on           Default: 3000
+`;
+
 
 if (import.meta.main) {
-  cli(Deno.args);
+  await cli(Deno.args);
 }
 
 export default async function cli(args) {
-  const version = "v0.14.0";
-  let stop = false;
-  const options = parse(args, {
-    boolean: ["serve", "init", "version", "dev", "help", "upgrade", "update"],
-    string: ["run", "port", "src", "dest", "location", "root", "config"],
-    alias: {
-      help: "h",
-      version: "v",
-    },
-    ["--"]: true,
-    unknown(option) {
-      if (option.startsWith("-")) {
-        console.log(`Unknown option: ${option}`);
-        stop = true;
-      }
-    },
-    default: {
-      root: Deno.cwd(),
-      config: "_config.js",
-    },
+  // the rest of the option parsing is handled within each command
+  let options = parse(args, {
+    boolean: ["help", "version"],
+    alias: {help: "h", version: "v"},
   });
 
-  const command = options._[0] || "build";
-
-  if (stop) {
-    console.log(`Run ${brightGreen("lume --help")} for usage information`);
-    console.log("");
+  // lume --version
+  if (options.version) {
+    console.log(`🔥lume ${version}`);
     return;
   }
 
   if (options._.length > 1) {
```

Comment:
We have to remove this from there, because `run` command need two values here: `lume run deploy`. I think this should be checked by every command individually (like the arguments).

Reply:
done